### PR TITLE
Google shopping homework

### PR DESCRIPTION
**Pull request by saywandeusher**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 3

Completeness Level (1-5): 3

What did you think of this deliverable?: Codes are working smoothly but stuck at question 5.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    count++;
+});
+
+console.log(count);
+
+
+//Q1 option 2
+var productsItems = products.items;
+function kindShoppingProduct(x) {
+  var kSP = 0;
```
Comment (scott88lee) [inline script.js]: Try to name your variables more verbosely. So when others or when you read the code 3 months down the road, it'll be immediately understandable

File: script.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+//Q2
+var productsItems = products.items;
+function availabilitybackorder(x) {
+  for (var i = 0; i < x.length; i++){
+    if (x[i].product.inventories[0].availability == "backorder"){
+      console.log(x[i].product.title);
+    }
+  }
+}
+console.log(availabilitybackorder(productsItems));
```
Comment (scott88lee) [inline script.js]: Just call the function availabilitybackorder(productItems));

There's no need to nest a console log inside a consolelog.

Comment (saywandeusher) [reply script.js]: Got it!

File: script.js (hunk 1 above)
Comment (saywandeusher) [reply script.js]: Noted! Thanks scott!

**Pull request closed without merging**